addons/website_sale/static/src/js/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -1,11 +1,206 @@
 /** @odoo-module **/
 
 import { WysiwygAdapterComponent } from '@website/components/wysiwyg_adapter/wysiwyg_adapter';
-import { patch } from '@web/legacy/js/core/utils';
+import { patch, Markup } from '@web/legacy/js/core/utils';
 
 // TODO this whole patch actually seems unnecessary. The bug it solved seems
 // to stay solved if this is removed. To investigate.
 patch(WysiwygAdapterComponent.prototype, 'website_sale_wysiwyg_adapter', {
+    /**
```

Comment:
why this code move?

Reply:
Because he was in a file that was not in the correct asset. There was a patch for adaptater in two differents files